src/mocks/msw/handlers.ts

```diff
@@ -1,274 +1,18 @@
-import {
-  BackendDestinationEnum,
-  getEndpoint,
-} from '../../lib/utils/functions/getEndpoint'
-import { mockSchemaBinding } from '../mockSchema'
 import { rest } from 'msw'
-import {
-  mockFileEntity,
-  mockFileEntityBundle,
-  mockProjectEntity,
-  mockProjectEntityBundle,
-  mockPaginatedEntityHeaders,
-  MOCK_FILE_ENTITY_ID,
-  MOCK_FILE_NAME,
-  MOCK_INVALID_PROJECT_NAME,
-  MOCK_PROJECT_ID,
-  MOCK_PROJECT_NAME,
-  mockFileEntityJson,
-  mockFileEntityVersionInfo,
-  mockFileEntityVersions,
-} from '../entity/mockEntity'
-import {
-  mockUserBundle,
-  mockUserProfileData,
-  MOCK_USER_ID,
-} from '../user/mock_user_profile'
-import {
-  ENTITY,
-  ENTITY_BUNDLE_V2,
-  ENTITY_SCHEMA_BINDING,
-  FAVORITES,
-  USER_ID_BUNDLE,
-  USER_PROFILE_ID,
-  USER_PROFILE,
-  ENTITY_JSON,
-  ENTITY_ID,
-  ENTITY_ID_VERSION,
-  ENTITY_ID_VERSIONS,
-} from '../../lib/utils/APIConstants'
-import {
-  Entity,
-  EntityBundle,
-  PaginatedResults,
-  UserBundle,
-  UserProfile,
-  VersionableEntity,
-} from '../../lib/utils/synapseTypes'
 import { SynapseError } from '../../lib/utils/SynapseClient'
-import { VersionInfo } from '../../lib/utils/synapseTypes/VersionInfo'
+import { entityHandlers } from './handlers/entityHandlers'
+import { userProfileHandlers } from './handlers/userProfileHandlers'
 
 // Simple utility type that just indicates that the response body could be an error like the Synapse backend may send.
-type SynapseApiResponse<T> = T | SynapseError
+export type SynapseApiResponse<T> = T | SynapseError
 
 const handlers = [
   rest.options('*', async (req, res, ctx) => {
     return res(ctx.status(200))
   }),
 
-  rest.get(
```

Comment:
Everything was just moved to entityHandlers.ts or userProfileHandlers.ts just to be a little more organized.